cmd/binarycookies/main.go

```diff
@@ -4,25 +4,40 @@ import (
 	"flag"
 	"fmt"
 	"os"
+	"regexp"
 
 	"github.com/cixtor/binarycookies"
 )
 
 var filename string
+var netscape bool
+var filter string
 
 func main() {
 	flag.Usage = func() {
-		fmt.Println("Usage: binarycookies [/path/to/Cookies.binarycookies]")
+		fmt.Println("Usage: binarycookies [-netscape] [-re regexp] [/path/to/Cookies.binarycookies]")
```

Comment:
The text says to use `-re [regexp]` but the flag is defined as `-filter`.

Reply:
Thanks for picking that up. :)